stockflux-securities-master/src/components/Alert.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Alert.css";
+
+const Alert = ({ messages, type, className }) => {
+  return (
+    <ul className={`securities-alert-container ${className}`}>
+      {messages.map((message, index) => {
+        return (
+          <li className={`securities-alert ${type}`} key={index}>
+            {message}
+          </li>
+        );
+      })}
+    </ul>
+  );
+};
+
```

Comment:
If you want to enforce that the `messages` parameter is an array so you don't have to do that check, you can use `propTypes` to ensure that it is an array of strings. You can even use `PropTypes.oneOf(...)` to specify that it should be either a string or an array of strings.

Reply:
Alright, I think I'll go with adding the check in like above as it makes it more versatile

Reply:
I'll also add the prop types in as you suggested

Reply:
done